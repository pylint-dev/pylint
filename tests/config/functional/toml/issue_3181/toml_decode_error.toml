# TOML decode error crash pylint
[tool.pylint]
***
