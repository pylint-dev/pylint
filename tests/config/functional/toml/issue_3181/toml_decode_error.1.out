************* Module {abspath}
{relpath}:1:0: F0011: error while parsing the configuration: Invalid statement (at line 3, column 1) (config-parse-error)
